dali/kernels/imgproc/convolution/cutlass/threadblock/mma_pipelined.h

```diff
@@ -173,13 +198,14 @@ class MmaPipelined : public MmaBase<Shape_, Policy_, 2> {
   /// Perform a threadblock-scoped matrix multiply-accumulate
   CUTLASS_DEVICE
   void operator()(
-      int gemm_k_iterations,                    ///< number of iterations of the mainloop
-      FragmentC &accum,                         ///< destination accumulator tile
-      IteratorA iterator_A,                     ///< iterator over A operand in global memory
-      IteratorB iterator_B,                     ///< iterator over B operand in global memory
-      FragmentC const &src_accum,               ///< source accumulator tile
-      TransformA transform_A = TransformA(),    ///< transformation applied to A fragment
-      TransformB transform_B = TransformB()) {  ///< transformation applied to B fragment
+      int gemm_k_iterations,                      ///< number of starting iteration of the mainloop
+      int end_iteration,                          ///< number of ending iteration of the mainloop
```

Comment:
```suggestion
      int skip_last_k_iterations,                 ///< number of ending iteration of the mainloop
```
?

Reply:
Also renamed as suggested in the other comment